data_symbols.syms

```diff
@@ -345,6 +345,9 @@ swrTextEntries1Count 0x0050c750 int # < 0x80
 
 swrTextEntries2Count 0x0050c758 int # < 0x20
 
+swrTextFmtString1 0x004b2304 char[4] = "%s\0\0"
+swrTextFmtString2 0x004c3e48 char[8] = "~f%d%s\0\0"
+
 rdMatrixStack34_size 0x0050c6f4 int
```

Comment:
@tim-tim707 this feels somewhat unnatural. I did this as in the decomp, it was pointing at DAT_ values with these strings present

However, in natural coding I would likely just type in the string as a constant. I tried to follow the decomp, but wanted to ask a style question when encountering these format strings (there are a number present)

Reply:
I dont really know what would be best here. I personnaly like righting the string constant like in C but it wont be in the right location in the executable afterward. I dont think its important since the data is the same and the location irrelevant for function. It will still make matching decomp harder later on. Maybe we can do for now and worry about matching later.

Reply:
Ok, so I think if that is the case, lets go with what I have here and I will follow this convention - we can clean up these globals when we are free from hooking in the future